editor/src/components/canvas/canvas-strategies/canvas-strategies.tsx

```diff
@@ -119,11 +120,27 @@ export function pickCanvasStateFromEditorStateWithMetadata(
   }
 }
 
+function insertionSubjectsFromInsertMode(subject: InsertionSubject): InsertionSubjects {
```

Comment:
nit: shouldn't this be `getUnsertionSubjectsFromInsertMode`?

Reply:
renamed it